### PR TITLE
Add PyLint to CircleCI.

### PR DESCRIPTION
**Pull request by TheJMaster**

### CODE REVIEW

**Comment by Jack-Khuu** [comment]
Lgtm

**Review by Jack-Khuu — APPROVED** [review]

File: test/test_main.py [hunk 1]
```diff
@@ -11,48 +11,53 @@
...
-        ops = f.readlines()
+
+        ops = test_file.readlines()
 
         # Run every client as subprocess.
         for client_num in range(num_clients):
             client_ops = [op for op in ops if "C{}:".format(client_num) in op]
-            clean_client_ops = list(map(clean_op, client_ops))
-            run_server(server_port, client_num, clean_client_ops)
+            _ = list(map(clean_op, client_ops))
```
Comment (LukasJoswiak) [inline test/test_main.py]: Is this line doing anything?

**Review by LukasJoswiak — CHANGES_REQUESTED** [review]
See comment.

File: test/test_main.py (hunk 1 above)
Comment (TheJMaster) [reply test/test_main.py]: Oh yeah, that can be removed.

**Review by LukasJoswiak — APPROVED** [review]

**Pull request merged**